spec/fixtures/files/collection.json

```diff
@@ -22,7 +22,8 @@
       },
       "workflow": {
         "ingest_origin": "AllSupportedMODS-TestForDev.xls",
-        "publishing_state": "published"
+        "publishing_state": "published",
+        "processing_state": "complete"
```

Comment:
Do we need to add this? My understanding was that we were going to leave this value blank for workflows for Institution and Collection objects.

(Same comment for the other collections and institution JSON fixtures.)